File: Data Structures & Algorithms/duplicate-integer/submission-1.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        int index  = 0;
        while(index < nums.length){
            for (int i = 0; i < nums.length; i++){
                if (i != index){
                    if (nums[i] == nums[index])
                        return true;
                }
            }
            index++;
        }
        return false;
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        map1 = {}
        for i, n in enumerate(nums): 
            x = target - n

            if (x in map1):
                return [map1[x], i]
            map1[n] = i